rspec-sidekiq.gemspec

```diff
@@ -11,7 +11,8 @@ Gem::Specification.new do |s|
   s.description = 'Simple testing of Sidekiq jobs via a collection of matchers and helpers'
   s.license     = 'MIT'
 
-  s.add_dependency 'rspec-core', '~> 3.0', '>= 3.0.0'
+  s.add_dependency 'rspec-core', '~> 3.0'
+  s.add_dependency 'rspec-expectations', '~> 3.0'
```

Comment:
This is for `include RSpec::Matchers::Composable` that provides `values_match?` argument matching, and [compound expectations](https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/compound-expectations).